modules/ocf_kubernetes/manifests/master/persistent_volume.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_kubernetes::master::persistent_volume {
+
+  file {
+    default:
+      require => Package['kubeadm', 'kubectl'];
```

Comment:
nit: While it surely doesn't hurt, I don't think `kubeadm` is required for any of this.